### ISSUE TITLE
Improve accessibility 

### ISSUE THREAD

**Issue by YuriyVaadin** [vaadin/vaadin-date-picker#580]
### Description
 - Make sure the `vaadin-date-picker` can be controlled as an input filed and be present in form field list of NVDA
   - NVDA navigation shortcuts should be applied to the `vaadin-date-picker` to focus / use the element directly. Like <kbd>Shift</kbd> + <kbd>E</kbd> (E as input field) [German: 'Eingabe Feld']
(Shortcuts examples: https://dequeuniversity.com/screenreaders/nvda-keyboard-shortcuts)
   - `vaadin-date-picker` should be labeled in form field list. Example of expected functionality can be found under "Advanced Usage" under "Custom Inputs", where NVDA uses the "placeholder" as the name of the form field. 
 - Navigation with arrow keys should be polished: date and the associated day of the week should be displayed in first place. Currently when navigating with the arrow key down, all seven days of the week are displayed first, followed by a date and the associated day of the week. 
 - Limit the amount of switches in NVDA switch list to a smaller amount (for example to a month, visible dates or a week). Currently there is an incredible amount of switches in NVDA switch list. Each date, e.g. 1 June 1980, Sunday, becomes the switch.
 - Hide extra elements from the lists. For example, the month has a headline and it appears in the heading list.
